ndb/src/google/cloud/ndb/async_.py

```diff
@@ -0,0 +1,52 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Manage asynchronous context. """
```

Comment:
I'm surprised `black` doesn't add an empty line in between the docstring and the first import

Reply:
It doesn't. I have PEP8 checker running in my editor, as well, and it doesn't care.  I can add one anyway.